### PR TITLE
Update item page claim and fact-check fields to CDS [CV2-2932]

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

- Update inputs on fact-check and claim to be CDS inputs
- Deprecate `MediaAnalysis.js` after confirming with @caiosba and @pierreconti 
- Fix minor nit where last request in list had bottom border
- Improve the placeholder copy for the claim and fact-check fields
- Remove the proceed dialog confirmation in favor of an alert box and disabling the button on fact-checks without claims
- Changed the fact-check URL from textarea to textfield since its a URL
- Add new global css classes for laying out forms
- Fix the border radius of menus so they match buttons

References: [CV2-2932](https://meedan.atlassian.net/browse/CV2-2935)

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can verify the changes. Please describe whether or not you implemented automated tests.

### DEPRECATED
Screenshot 2023-10-16 at 2 00 09 PM

### BEFORE - menu corners showing button behind
Screenshot 2023-10-17 at 9 08 55 AM

### AFTER - menu corners
Screenshot 2023-10-17 at 9 09 23 AM

### AFTER - last request card no bottom border
Screenshot 2023-10-19 at 2 45 35 PM

### AFTER - empty claim and fact-check
Screenshot 2023-10-19 at 2 46 09 PM

### AFTER - filled claim and fact-check
Screenshot 2023-10-19 at 2 47 15 PM

### AFTER - errors moved to be in context
Screenshot 2023-10-19 at 2 47 49 PM

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

[CV2-2932]: https://meedan.atlassian.net/browse/CV2-2932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by brianfleming** [comment]
Note to reviewers. The failure of the last build are similarity which I believe is unrelated to this branch. integration and unit test pass

File: src/app/components/layout/inputs/LimitedTextArea.js [hunk 1]
```diff
@@ -53,13 +54,15 @@ const LimitedTextArea = ({
...
       helpContent={(
         <>
-          {helpContent && (<>{helpContent}<br /></>)}
-          <FormattedMessage
-            id="limitedTextAreaWithCounter.counter"
-            defaultMessage="{remaining, plural, one {# character left} other {# characters left}}"
-            description="Label that displays how many characters more can be typed"
-            values={{ remaining: maxChars - (localText.length || 0) }}
-          />
+          {helpContent && (<>{helpContent}</>)}
```
Comment (amoedoamorim) [inline src/app/components/layout/inputs/LimitedTextArea.js:57]: Could this line simply become `{helpContent}` now? :smile:

Comment (brianfleming) [reply src/app/components/layout/inputs/LimitedTextArea.js]: ha ha...what's wrong @amoedoamorim ?? you don't like to be really really sure about things??
fixed!

File: src/app/components/media/MediaFactCheck.js [hunk 2]
```diff
@@ -154,30 +154,39 @@ const MediaFactCheck = ({ projectMedia }) => {
...
-      <Box id="media__fact-check-title" display="flex" alignItems="center" mb={2} mt={2} justifyContent="space-between">
-        <div className="typography-subtitle2">
-          <FormattedMessage id="mediaFactCheck.factCheck" defaultMessage="Fact-check" description="Title of the media fact-check section." />
-        </div>
-        {' '}
-        <div className="typography-caption">
-          { error ? errorMessage : null }
-          { saving && !error ?
+    <div id="media__fact-check" className={cx(styles['media-item-claim-inner'], inputStyles['form-fieldset'])}>
+      { (!claimDescription || claimDescription.description?.trim()?.length === 0) ?
```
Comment (amoedoamorim) [inline src/app/components/media/MediaFactCheck.js:159]: Can we have a this as a variable named `claimDescriptionMissing` or something?

File: src/app/components/media/MediaFactCheck.js [hunk 3]
```diff
@@ -188,81 +197,126 @@ const MediaFactCheck = ({ projectMedia }) => {
...
+        <div className={inputStyles['form-fieldset-field']}>
           <ButtonMain
             onClick={handleGoToReport}
             className="media-fact-check__report-designer"
             variant="contained"
             theme={published ? 'brand' : 'alert'}
             size="default"
             iconLeft={published ? <IconReport /> : <IconUnpublishedReport />}
-            disabled={saving || readOnly}
+            disabled={saving || readOnly || (!claimDescription || claimDescription.description?.trim()?.length === 0)}
```
Comment (amoedoamorim) [inline src/app/components/media/MediaFactCheck.js:319]: and reuse that variable named `claimDescriptionMissing` here? :D

**Review by amoedoamorim — APPROVED** [review]

File: src/app/components/media/MediaFactCheck.js (hunk 3 above)
Comment (brianfleming) [reply src/app/components/media/MediaFactCheck.js]: Great idea @amoedoamorim I have addressed this. Thanks for the great review overall

**Pull request merged**